hphp/runtime/vm/jit/vasm-simplify-arm.cpp

```diff
@@ -99,6 +99,25 @@ bool simplify(Env& env, const loadb& inst, Vlabel b, size_t i) {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+bool simplify(Env& env, const ldimmq& inst, Vlabel b, size_t i) {
+  return if_inst<Vinstr::lea>(env, b, i + 1, [&] (const lea& ea) {
+    // ldimmq{s, tmp}; lea{tmp, d} -> subqi{s, d}
```

Comment:
This description makes no sense.  `ldimmq{}` has a Vreg dst, while `lea{}` has a Vptr src; those are incompatible.

Reply:
Please update this as follows:

```cpp
// ldimmq{s, index}; lea{base[index], d} -> lea{base[s], d}
```